### DISCUSSION TITLE
强烈建议repo主立即做好安全准备！

### DISCUSSION THREAD

**Discussion by yalo-ddaa** [WeNeedHome/SummaryOfLoanSuspension#283] [Ideas]
希望还没晚。
如果号主在国内的话，建议赶快联系一些安全的人，给予仓库修改权限，并立即加密自己电脑和手机上的敏感数据，账户密码不要使用常用或相似密码！
不要使用任何国内APP（甚至是手机等设备）讨论有关事宜，以免被有关单位定位。也不要随便乱加入别人发的tg群链接，以免存在内鬼。

之前996icu项目就是因为号主失踪，结果这个仓库之后就再也没更新过，希望不会重蹈覆辙。留得青山在，不怕没柴烧。

最重要的是，如果上面发现抓一个人没意义，也许就不会真的行动。所以这样应该更能对repo主起到保护作用。

如果大家还有其他建议的话欢迎补充。

**Comment by Leon0312** [comment]
大家都fork 就永远封不完。

**Reply by yalo-ddaa** [reply]
不光是fork能不能封完的问题，而是组织度的问题。

**Reply by yalo-ddaa** [reply]
fork是无组织的支持，而实际行动必须要有人进行组织。一旦失去组织，那剩下来的fork也只是想想而已，起不到关键作用。
所以维稳才会经常请那些组织者喝茶。但组织者不一定得是一个人，所以我建议repo主可以给其他安全并且信得过的人修改权限，以此分摊风险。

**Reply by gds-970423** [reply]
好家伙  我直呼好家伙

**Comment by DSbajRv5Ol4X** [comment]
留个电报群吧，然后其他注册电报别用国内手机号，网上买个境外旅游手机卡，用户名要随机，输入法，还有翻墙工具别用如老王这种。

**Reply by MaiXiaochai** [reply]
电报前段时间不是被拿捏了吗？忘了是英国警方还是老美，现在它们可以访问电报敏感数据了。输入法有推荐的吗？我平时都不给输入法各种想获取我信息或者联网的权限，估计也不安全。

**Reply by yalo-ddaa** [reply]
> 电报前段时间不是被拿捏了吗？忘了是英国警方还是老美，现在它们可以访问电报敏感数据了。输入法有推荐的吗？我平时都不给输入法各种想获取我信息或者联网的权限，估计也不安全。

但telegram肯定比大多数app安全的多。

**Comment by ke-qiong** [comment]
996icu是号主失踪，是因为被抓了吗，还是仅仅是警告

**Reply by yalo-ddaa** [reply]
失踪了，所以不知道。

**Reply by kaizibaby** [reply]
我靠真嗐人

**Comment by Ice-Storm** [comment]
可以把内容放到IPFS上，但是现在缺少一种基于去中心化的编辑协同工具

**Comment by O-kai** [comment]
我党做事的风格从来就是，解决不了问题，就把提出问题的人解决掉。🤦‍♂️🤦‍♂️🤦‍♂️

**Reply by DDFighting** [reply]
治不了烂尾楼，我还治不了你了？.png
